_includes/_meta_information.html

```diff
@@ -12,7 +12,7 @@
 				<time class="icon-calendar pr20" datetime="{{ page.date | date: "%Y-%m-%d" }}" itemprop="datePublished"> {{ page.date | date: "%Y-%m-%d" }}</time>
 				{% endif %}
 
-				{% if page.categories %}<span class="icon-archive pr20"> {{ page.categories | join: ' · ' | upcase }}{% endif %}</span>
+				{% if page.categories and page.categories.size > 0 %}<span class="icon-archive pr20"> {{ page.categories | join: ' · ' | upcase }}{% endif %}</span>
```

Comment:
The first one reminds me of a certain John Resig hack for jQuery. Nice pith there behind door #2. Yoink.